.github/workflows/pypipublish_linux.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
       - name: Build pydp lib
         run: |
-          bazel build --config Linux src/python:bindings_test  --verbose_failures
+          bazel build --config ${{ matrix.os }} src/python:pydp  --verbose_failures
```

Comment:
I am not sure if `matrix.os` would pick OS since this is hardcoded. Is this verified?

Reply:
I have actually refactored these now that the tests pass anyway, so its going to be identical to the existing tests yaml which works.